client/my-sites/checkout/src/components/payment-method-logos.js

```diff
@@ -3,19 +3,19 @@ import styled from '@emotion/styled';
 export const PaymentMethodLogos = styled.span`
 	display: flex;
 	flex: 1;
-	text-align: right;
+	flex-wrap: wrap;
+	gap: 5px;
+	text-align: end;
 	align-items: center;
-	justify-content: flex-end;
+	justify-content: flex-start;
 
-	.rtl & {
-		text-align: left;
-	}
-
-	svg {
+	&.credit-card__logo svg {
 		display: inline-block;
+		height: 26px;
+		width: 42px;
+	}
 
-		&.has-background {
```

Comment:
Not really needed if we set a default size for the card logos and apply some flex gap.